addons/mail/models/ir_http.py

```diff
@@ -13,7 +13,14 @@ def session_info(self):
         """Override to add the current user data (partner or guest) if applicable."""
         result = super().session_info()
         store = Store()
-        self.env["res.users"]._init_store_data(store)
+        ResUsers = self.env["res.users"]
+        if cids := request.httprequest.cookies.get("cids", False):
```

Comment:
There already is a default value for [`get`](https://docs.python.org/3/library/stdtypes.html#dict.get), you don't need to specify one. Also you see that we assign `False` on recordsets (for null) and that we return `False` in some public methods, it is for compatibility with XMLRPC. `None` is much more appropriate in this case.

Just spreading the good practices, no need to change it.